model/src/main/java/org/cbioportal/model/GenesetAlteration.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class GenesetAlteration implements Serializable {
```

Comment:
This is the `genetic_alteration` joined with `geneset` table, and current `GeneticAlteration.java` is the `genetic_alteration` joined with `gene` table, can you rename the `GeneticAlteration.java` to `GeneAlteration.java` to be consistent with this one, and then create an abstract parent class for both of them named `GeneticAlteration.java` (won't be used anywhere, just to indicate those classes are related, uses the same table), which will have `geneticEntityId` and `values`, and then remove `values` from both of those classes?